apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/EvaluationEditor/Editor/Playground/Variables/index.tsx

```diff
@@ -0,0 +1,273 @@
+'use client'
+
+import { ReactNode, useEffect, useState } from 'react'
+
+import { Config, readMetadata } from '@latitude-data/compiler'
+import { ProviderLogDto } from '@latitude-data/core/browser'
+import {
+  formatContext,
+  formatConversation,
+} from '@latitude-data/core/services/providerLogs/formatForEvaluation'
+import {
+  Badge,
+  Button,
+  cn,
+  CodeBlock,
+  CollapsibleBox,
+  Icon,
+  Popover,
+  Text,
+  Tooltip,
+} from '@latitude-data/web-ui'
+import useDocumentLogWithMetadata from '$/stores/documentLogWithMetadata'
+
+const PARAMETERS = [
+  'messages',
+  'context',
+  'response',
+  'prompt',
+  'parameters',
+  'config',
+  'duration',
+  'cost',
+]
```

Comment:
Move this to constants or something more shared. Ideally we should have these only defined once, otherwise adding more parameters to it will be a pita.